packages/components/addon/components/hds/tabs/index.js

```diff
@@ -49,6 +49,13 @@ export default class HdsTabsIndexComponent extends Component {
   handleClick(tabIndex) {
     this.selectedTabIndex = tabIndex;
     this.setTabIndicator(tabIndex);
+
+    // Scroll Tab into view if it's out of view
+    this.tabNodes[tabIndex].parentNode.scrollIntoView({
+      behavior: 'smooth',
+      block: 'nearest',
+      inline: 'nearest',
+    });
```

Comment:
I added this so if a user clicks on a tab which is partially offscreen, the full tab will scroll into view. Let me know if you don't think this should be included in the basic `Tabs` or could use modifications though. (This behavior was part of the specified overflow nav behavior but I thought it would make sense included in the basic Tabs.)

https://hds-components-git-hds-450-initial-polish-hashicorp.vercel.app/components/tabs#showcase

Reply:
👏👏👏👏👏👏👏👏👏👏👏

Reply:
Thanks!